README.zh-cn.md

```diff
@@ -0,0 +1,195 @@
+[![#yourfirstpr](https://img.shields.io/badge/first--timers--only-friendly-blue.svg)](https://github.com/nanoframework/Home/blob/main/CONTRIBUTING.md) [![Discord](https://img.shields.io/discord/478725473862549535.svg?logo=discord&logoColor=white&label=Discord&color=7289DA)](https://discord.gg/gCyBu8T)
+
+![nanoFramework logo](https://github.com/nanoframework/Home/blob/main/resources/logo/nanoFramework-repo-logo.png)
+
+-----
+文档语言: [English](README.md) | [简体中文](README.zh-cn.md)
+
+# 欢迎使用.net **nanoFramework** IoT.Device Library repository
+
+该存储库包含绑定，可以是传感器，小屏幕和任何您可以连接到纳米框架芯片的东西!
+
+大多数绑定都是从[.NET IoT repository](https://github.com/dotnet/iot/tree/main/src/devices)迁移过来的。并不是所有的绑定都对迁移到.net纳米框架有意义，所以迁移的工作已经放在了能够与.net nanoFramework 工作的设备上。也请注意，一些设备已经迁移，没有测试，所以它们主要包含问题。
+
+## 设备列表
+
+<devices>
```

Comment:
all this part is generated automatically. So you should adjust the code from the application generating this part.
Here: https://github.com/nanoframework/nanoFramework.IoT.Device/tree/develop/src/device-listing

So adjust the code to generate this section from the device listing code.